src/Repository/RevisionRepository.php

```diff
@@ -309,6 +309,24 @@ public function findInProgresByContentType($contentType, $circles, $isAdmin)
         return $qb->getQuery()->getResult();
     }
 
+    /**
+     * @return iterable|Revision[]
+     */
+    public function findAllDraftsByContentTypeName(string $contentTypeName): iterable
+    {
+        $qb = $this->createQueryBuilder('r');
+        $qb
+            ->join('r.contentType', 'c')
+            ->andWhere($qb->expr()->isNull('r.endTime'))
+            ->andWhere($qb->expr()->eq('r.draft', $qb->expr()->literal(true)))
```

Comment:
You missed a r.deleted = :false. Or not?

Reply:
👍